### PR TITLE
Bug 1123184 - Enable l10n args format in browser context menu

### PR DESCRIPTION
**Pull request by shamenchens**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[shamenchens Sherman Chen (shamenchens)](https://github.com/shamenchens) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1e2b4aaed5d2)

File: tv_apps/app-deck/js/context_menu.js [hunk 1]
```diff
@@ -121,11 +121,31 @@
       }
     },
 
+    _localizeElement: function cm_localizeElement(node, payload) {
+      if (typeof payload === 'string') {
+        node.setAttribute('data-l10n-id', payload);
+        return;
+      }
+
+      if (typeof payload === 'object') {
```
Comment (huchengtw-moz) [inline tv_apps/app-deck/js/context_menu.js]: according to logic, please use `else if` here.

File: tv_apps/app-deck/js/context_menu.js [hunk 2]
```diff
@@ -121,11 +121,31 @@
...
+        if (payload.id) {
+          navigator.mozL10n.setAttribute(node, payload.id, payload.args);
+          return;
+        }
+
+        if (payload.raw) {
```
Comment (huchengtw-moz) [inline tv_apps/app-deck/js/context_menu.js]: the same, `else if` here.

File: tv_apps/smart-system/js/browser_context_menu.js [hunk 3]
```diff
@@ -165,6 +165,26 @@
     this.scrollable.catchFocus();
   },
 
+  BrowserContextMenu.prototype._localizeElement = function(node, payload) {
```
Comment (huchengtw-moz) [inline tv_apps/smart-system/js/browser_context_menu.js]: Since _localizeElement is the same as app_deck, how about move this function to `SharedUtils`.

**Comment by try-server-hook** [comment]
[shamenchens Sherman Chen (shamenchens)](https://github.com/shamenchens) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=57d0f27195be)

**Comment by try-server-hook** [comment]
[shamenchens Sherman Chen (shamenchens)](https://github.com/shamenchens) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=bd9d7686aa59)

**Pull request merged**